src/common/components/hosts/tableUtils.tsx

```diff
@@ -486,20 +486,23 @@ export const hostActionResolver = ({
     if (canUnbindHost) {
       // skip at all if the callback is not provided
       const canUnbindHostResult = canUnbindHost(host);
-      const isDisabled = !canUnbindHostResult?.[0];
-
-      actions.push({
-        title: (
-          <ActionTitle
-            disabled={isDisabled}
-            description={canUnbindHostResult?.[1]}
-            title="Remove from the cluster"
-          />
-        ),
-        id: `button-unbind-host-${hostname}`,
-        onClick: () => !isDisabled && onUnbindHost && onUnbindHost(host),
-        disabled: isDisabled,
-      });
+      const isEnabled = canUnbindHostResult?.[0];
+      const disabledReason = canUnbindHostResult?.[1];
```

Comment:
`canUnbindHost` should always return the `[boolean, string]` tuple, right?
```
const [isEnabled, disabledReason] = canUnbindHostResult(host);
```

Reply:
right, changed